src/richie-front/js/components/CourseGlimpse/CourseGlimpse.tsx

```diff
@@ -35,22 +35,25 @@ export const CourseGlimpse = injectIntl(
     const { course, intl, organizationMain } = props;
 
     return (
-      <div className="course-glimpse-container">
-        <div className="course-glimpse">
+      <a className="course-glimpses__item course-glimpses__item--link" href="#">
+        <div className="course-glimpses__item__media">
           <img
-            className="course-glimpse__image"
             src={'https://www.fun-mooc.fr' + course.thumbnails.small}
             alt={intl.formatMessage(messages.altText, {
               courseTitle: course.title,
             })}
           />
-          <div className="course-glimpse__body">
-            <div className="course-glimpse__body__title">{course.title}</div>
-            <div className="course-glimpse__body__org">
+        </div>
+        <div className="course-glimpses__item__content">
+          <div className="course-glimpses__item__wrapper">
```

Comment:
The name for this class does not look consistent with whats above and below. I don't understand the styling well enough to judge the necessity of this wrapper but it should probably be named `course-glimpse__content__wrapper` (and maybe the inner items can keep `course-glimpse__content__title` etc. for the sake of simplicity and understandability).